pkg/steps/new_e2e.go

```diff
@@ -66,14 +75,20 @@ func (s *newE2ETestStep) Run(ctx context.Context, dry bool) error {
 		return nil
 	}
 	var errs []error
-	if err := s.runSteps(ctx, s.pre); err != nil {
+	var err error
+	if err = s.runSteps(ctx, s.pre, ""); err != nil {
```

Comment:
Don't we need the secret in the pre-steps so the installer can commit the KUBECONFIG to it in the first place?

Reply:
The main reason for repurposing artifacts is because the `installer` image doesn't have `oc` in it. If we can enforce that `pre` steps must be able to modify a secret, I'll gladly remove this.